pyproject.toml

```diff
@@ -21,11 +21,43 @@ disable = [
   "W1113",  # Keyword argument before variable positional arguments  (PEP-570 is Python 3.8)
 ]
 
+[tool.ruff]
```

Comment:
Reading https://lwn.net/Articles/930487/ and adding ruff to my own Python projects, I was wondering. Should we in the future extend the selected rules? I noticed ruff has a **ton** of rules!

Reply:
The idea of this PR was to get things online without too many changes :sweat_smile: